### ISSUE TITLE
Polaris factioned asteroid belts are empty

### ISSUE THREAD

**Issue by michael-harris** [RavienGaming/Reforged-Eden-2#87]
### Reforged Eden 2 Build Number

8

### Game mode

Survival, Dedicated Server

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

The polaris belts around stars in polaris space have no asteroids.  I will check outside polaris space if I see a polaris belt and update the issue.

**Comment by michael-harris** [comment]
Updated title as I have verified that these belts are also empty OUTSIDE polaris space as well.

**Comment by RavienGaming** [comment]
Please list some of the star types and playfield types this is happening in IE: Polaris Mining Site, Polaris Asteroid Field, K, G, F, etc.

**Comment by michael-harris** [comment]
Polaris Asteroid Field.  If you look at the info for "orbit" in the system map you'll see they have no asteroid types listed at all.  This has been true of every single one that I've seen around every star type.  In fact, I just noticed one of the regular asteroid belts showed up that way as well.  I did not make a note of the star type for that one though.  I am not actually sure if they are really empty or just not showing the Asteroid types in the info pane.  I can check that next time I see one.
